topics/wallet-standard.md

```diff
@@ -0,0 +1,248 @@
+# Wallet Standard
```

Comment:
I would call this a "Wallet _Message_ Standard" (or similar), since it only deals with this one aspect of wallets.

Reply:
I also took the liberty of [reserving an ICRC number (ICRC-25)](https://github.com/dfinity/ICRC/issues/29) for this specification. Please include it in the title.

Reply:
Update 4: We agreed on naming it Wallet Interaction Standard.